openwisp_notifications/base/models.py

```diff
@@ -84,22 +84,19 @@ def _get_related_object_url(self, field):
             if url:
                 try:
                     url_callable = import_string(url)
-                    return url_callable(self, field=field, absolute_url=True)
+                    return url_callable(self, field=field, absolute_url=absolute_url)
                 except ImportError:
                     return url
-        return _get_object_link(self, field=field, absolute_url=True)
+        return _get_object_link(self, field=field, absolute_url=absolute_url)
 
-    @property
-    def actor_url(self):
-        return self._get_related_object_url(field='actor')
+    def actor_url(self, absolute_url=False):
+        return self._get_related_object_url(field='actor', absolute_url=absolute_url)
 
-    @property
-    def action_url(self):
-        return self._get_related_object_url(field='action_object')
+    def action_url(self, absolute_url=False):
+        return self._get_related_object_url(field='action_object', absolute_url=absolute_url)
 
-    @property
-    def target_url(self):
-        return self._get_related_object_url(field='target')
+    def target_url(self, absolute_url=False):
```

Comment:
If we turn these properties into methods which support passing arguments we have to change their name to `get_{property_name}`, for backward compatibility we can keep the properties and internally call the new method which passes `absolute_url=True`.